.github/workflows/release.yml

```diff
@@ -0,0 +1,44 @@
+name: release
+
+on:
+  push:
+    tags:
+      - "v*"
+
+jobs:
+  test:
+    runs-on: ubuntu-20.04
+    env:
+      AQUASCOPE_SERVER: "https://aquascope.willcrichton.net"
+    steps:
+      - uses: actions/checkout@v2
+      - name: Standalone connect
+        run: |
+          cd frontend
+          graco build -w &
+          graco test
+
+  publish-standalone-and-docs:
+    needs: test
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v1
+      - run: npm install -g graco && cd frontend && graco prepare
+      - run: cargo doc --lib && mv ./target/doc ./frontend/packages/aquascope-standalone/dist/doc
+      - uses: JamesIves/github-pages-deploy-action@v4.3.0
+        with:
+          branch: gh-pages
+          folder: ./frontend/packages/aquascope-standalone/dist
+          clean: true
+          clean-exclude: dev
+
+  publish-crate:
+    needs: publish-standalone-and-docs
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v1
+      - run: cargo login ${{ secrets.CRATES_IO_TOKEN }} && cargo install cargo-workspaces
```

Comment:
Let's use the install action with `tool-cache: true` so we might avoid the build cost for `cargo-workspaces`.